muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/navbar/language-picker.tsx

```diff
@@ -22,7 +22,7 @@ interface LanguagePickerState {
 
 class LanguagePicker extends React.Component<LanguagePickerProps, LanguagePickerState> {
   render(){
-    return <Dropdown modifier="language-picker" items={this.props.locales.avaliable.map((locale)=>{
+    return <Dropdown modifier="language-picker" items={this.props.locales.AVAILABLE.map((locale)=>{
```

Comment:
This would have caused a crash, now it is lowercase in one place and uppercase in the reducer.